pkg/testing/directmessaging_mock.go

```diff
@@ -1,4 +1,15 @@
-// Code generated by mockery v1.0.0. DO NOT EDIT.
```

Comment:
is this file autogenerated ? Why is it being modified then?

Reply:
It was originally auto-generated, but over the course of time people had been adding patches manually.